webapp/tests/karma/ts/services/enketo.service.spec.ts

```diff
@@ -156,14 +161,13 @@ describe('Enketo service', () => {
           }
         },
         { provide: ZScoreService, useValue: zScoreService },
+        { provide: CHTScriptApiService, useValue: chtScriptApiService },
         { provide: TransitionsService, useValue: transitionsService },
         { provide: TranslateService, useValue: translateService },
         { provide: FeedbackService, useValue: feedbackService },
       ],
     });
 
-    service = TestBed.inject(EnketoService);
```

Comment:
This was being constructed too soon, and the enketo service constructor was calling `init()` which meant services were being called before the tests had a chance to stub them. By moving this down into the test we can stub them before they get initialised.